src/basic/calendarspec.c

```diff
@@ -919,6 +926,23 @@ int calendar_spec_from_string(const char *p, CalendarSpec **spec) {
                 if (IN_SET(j, 0, 1)) {
                         p = strndupa(p, e - p - 1);
                         c->dst = j;
+                } else {
+                        char *last_space;
```

Comment:
`const char *`.

Reply:
@keszybz `strrchr` does not return a `const char *`.

Reply:
The cast from `char *` to `const char*` is implicit.

Reply:
Right, if both are `const` it's fine, otherwice `gcc` emits a warning. Fixed as well.